src/components/api.js

```diff
@@ -0,0 +1,16 @@
+async function getMemes() {
+  const response = await fetch('https://api.imgflip.com/get_memes');
+  const results = await response.json();
+  // memes is an object in data, is an object in API callback
+  const memes = results.data.memes;
+  const memesImages = memes.map(meme => {
+    return meme.url;
+  });
+  return memesImages;
+}
+
+export async function generateRandomMeme() {
```

Comment:
`getMemes` returns all memes, `generateRandomMeme` returns a random meme.
I would use the same naming convention:

```suggestion
export async function getRandomMeme() {
```